functions.php

```diff
@@ -6,7 +6,7 @@ function scripts_and_styles_method() {
   // library.js is to bundle plugins. my.js is your scripts. enqueue more files as needed
   $jslib = $templateuri."library.js";
   wp_enqueue_script( 'jslib', $jslib,'','',true);
-  $myscripts = $templateuri."main.js";
+  $myscripts = $templateuri."main.min.js";
```

Comment:
debugging doesn't work well with min+sourcemap
if u wanna inspect thru the consoe it sux cuz  dont know the names of vars so u r lost
debugger will usually print next to each line the value of the vars used on that line, that doesn't work either